app.js

```diff
@@ -0,0 +1,37 @@
+const express = require('express');
+const path = require('path');
+const bodyParser = require('body-parser');
+const morgan = require('morgan');		// Logs HTTP Requests in the server console
+const app = express();
+
+const add = require('./app/calculator');
+
+/* ============== SETUP ============== */
+
+app.use(morgan('dev'));
+app.use(bodyParser.urlencoded({ extended: false }));
+
+app.set('view engine', 'pug');
+app.set('views', path.join(__dirname, 'views'));
+
+/* ============== ROUTING =========== */
+
+app.get('/', (req, res) => {
```

Comment:
I think is good idea separate the routes in deferent files, also into a folder /routes/calculator.js

Reply:
I think is a great idea!!